src/main/kotlin/com/ecwid/apiclient/v3/dto/product/request/UpdatedProduct.kt

```diff
@@ -269,8 +269,8 @@ data class UpdatedProduct(
 	data class ProductOptionChoice(
 		val text: String = "",
```

Comment:
Why is this field still required?

Reply:
We need option choice name to both create a new option choice, and identify which one to update. So this field is functionally an id.